src/cachinglayer.ts

```diff
@@ -120,6 +122,21 @@ abstract class CachingLayer {
 
   abstract setNodes(nodes: RSNodes): Promise<void>;
 
+  /**
+   * Broadcast channel, used to inform other tabs about change events
+   */
+  broadcastChannel: BroadcastChannel;
+
+  constructor () {
+    const env = new Env();
+    if (env.isBrowser() && !!globalContext["BroadcastChannel"]) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/BroadcastChannel notes that BroadcastChannel is available in Node.js since v18 and Deno since 1.11.  Do we need to confine this to browsers?